automation/docker/Dockerfile

```diff
@@ -68,6 +69,27 @@ RUN curl -L https://dl.google.com/android/repository/sdk-tools-linux-${ANDROID_S
     && mkdir -p /build/android-sdk/.android/ \
     && touch /build/android-sdk/.android/repositories.cfg \
     && yes | sdkmanager --licenses
+ 
+#----------------------------------------------------------------------------------------------------------------------
+#-- Test tools --------------------------------------------------------------------------------------------------------
+#----------------------------------------------------------------------------------------------------------------------
+
+RUN mkdir -p /build/test-tools
+
+WORKDIR /build
+
+ENV TEST_TOOLS /build/test-tools
+ENV PATH ${PATH}:${TEST_TOOLS}:${TEST_TOOLS}/google-cloud-sdk/bin
+
+RUN curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GOOGLE_SDK_VERSION}.0.0-linux-x86_64.tar.gz --output ./gcloud.tar.gz \
+    && tar -xvf ./gcloud.tar.gz -C ${TEST_TOOLS} \
+    && rm -f ${GOOGLE_SDK_DOWNLOAD} \
```

Comment:
Nit: Undefined variable. I guess it refers to `./gcloud.tar.gz`?

Reply:
thanks, @JohanLorenzo,  done!  without the tar file, google cloud sdk adds 1G.